vpn-tunnel-modify.py

```diff
@@ -0,0 +1,38 @@
+#!/usr/bin/python
+
+import logging
+import boto3
+import os
+from random import *
+import json
+
+ec2 = boto3.client('ec2')
+
+
+logger = logging.getLogger()
+logger.setLevel(logging.INFO)
+
+
+def modify_vpn_tunnel(vpn_connection_id: str, vpn_tunnel_outside_ip_address: str) -> None:
+    logger.info("modify vpn tunnel")
+    random_dpd_timeout = randint(30, 130)  # Pick a random number between 30 and 130.
+
+    ec2.modify_vpn_tunnel_options(
+        VpnConnectionId=vpn_connection_id,
+        VpnTunnelOutsideIpAddress=vpn_tunnel_outside_ip_address,
+        TunnelOptions={
+            'DPDTimeoutSeconds': random_dpd_timeout,
```

Comment:
I don't think randomly changing the timeout is going to be an option, this will produce drift through the terraform config and DPD timeout might actually become a config that needs to be set to a precise value. - I'm still approving the PR though, we can always come back and update this later